gammapy/background/reflected.py

```diff
@@ -285,21 +285,34 @@ class ReflectedRegionsBackgroundEstimator:
 
     For a usage example see :gp-notebook:`spectrum_analysis`
 
+    TODO: make the reference map (to make the mask used to select events)
     Parameters
     ----------
-    on_region : `~regions.CircleSkyRegion`
-        Target region
+    on_region : `~regions.SkyRegion`
+        Target region with any shape, except `~region.PolygonSkyRegion`
     observations : `~gammapy.data.Observations`
         Observations to process
     kwargs : dict
-        Forwarded to `gammapy.background.ReflectedRegionsFinder`
+        Forwarded to `~gammapy.background.ReflectedRegionsFinder`
     """
 
     def __init__(self, on_region, observations, **kwargs):
         self.on_region = on_region
         self.observations = observations
-        self.finder = ReflectedRegionsFinder(region=on_region, center=None, **kwargs)
 
+        # Proper management of the bin size (as it has impact on results)
+        binsz = 0.01 * u.deg
```

Comment:
Is the impact connected to mask based region rejection or is it connected to the event selection once regions have been selected?

Reply:
both!